message.go

```diff
@@ -12,10 +12,24 @@ type Message struct {
 	TimeToLive            int                    `json:"time_to_live,omitempty"`
 	RestrictedPackageName string                 `json:"restricted_package_name,omitempty"`
 	DryRun                bool                   `json:"dry_run,omitempty"`
+	Notification          *Notification           `json:"notification"`
+}
+
+type Notification struct {
+	Title string `json:"title"`
+	Body  string `json:"body"`
 }
 
 // NewMessage returns a new Message with the specified payload
 // and registration IDs.
+// @DEPRECATED as no validation here and client should create itself freely
```

Comment:
Not sure I understand this comment

Reply:
Imo, we can leave the client or caller create the Message themselves. With that, they freely use or not use other fields ex: badge, sound .... in future.

Reply:
Does it mean that we will have some new code in our backend for creation of new message?

Reply:
exactly what I 'd like to do